example/yarn.lock

```diff
@@ -987,6 +987,19 @@
     plist "^3.0.1"
     xcode "^2.0.0"
 
+"@react-native-community/cli-platform-ios@^5.0.2":
```

Comment:
You might want to dedupe this. We have two versions in here now. Or just regenerate `yarn.lock` from scratch.